### PR TITLE
Move translation navigation under the main h1 on the transition and EU business landing pages

### PR DESCRIPTION
**Pull request by owenatgov**
## What
Moves the translation navigation component below the H1 on the [transition landing page](https://www.gov.uk/transition) and the [EU business page](https://www.gov.uk/eubusiness).

## Why
This ensures that instances of our translation navigation across gov.uk are compliant with [WCAG 3.2.3](https://www.w3.org/WAI/WCAG21/Understanding/consistent-navigation.html). Currently, the majority of translation nav components sit broadly below the H1 so this change ensures that gov.uk is consistent and therefore passes 3.2.3.

### Note
The brexit team still need to let No. 10 know this is changing to minimise any surprises causing problems so this is review only until they've given the goahead.

[Card](https://trello.com/c/Qa5BazvX/439-inconsistently-placed-translation-navigation)

## Visual changes

| Before | After |
| --- | --- |
| Screenshot 2021-04-13 at 15 46 55 | Screenshot 2021-04-13 at 15 48 01 |
| Screenshot 2021-04-13 at 17 34 20 | Screenshot 2021-04-13 at 17 34 34 |

### CODE REVIEW

**Comment by maxgds** [comment]
Looks ok to me, obviously I can't approve the PR at this stage, though. 
Is it worth getting a designer to ok it as well? It's probably fine but I'm not sure how we generally handle those key line/underline things in cases like this.

**Comment by miaallers** [comment]
Looks good to me!

**Comment by maxgds** [comment]
> Looks good to me!

If Mia's happy, I'm happy! Approved in as much as it can be ahead of sign-off.

**Comment by owenatgov** [comment]
Un-drafting this so that we can get the approval tick in prep for proposed deployment tomorrow.

**Review by sihugh — APPROVED** [review]
Merge when you've been given the green light. (Sorry for the faff)

**Pull request merged**
